.travis.yml

```diff
@@ -60,23 +60,19 @@ matrix:
         apt:
           packages:
           - xsel
-
-    # In allow_failures
     - dist: trusty
       env:
-        - JOB="3.6, slow" ENV_FILE="ci/deps/travis-36-slow.yaml" PATTERN="slow"
+        - JOB="3.6, doc" ENV_FILE="ci/deps/travis-36-doc.yaml" DOC=true
 
     # In allow_failures
     - dist: trusty
       env:
-        - JOB="3.6, doc" ENV_FILE="ci/deps/travis-36-doc.yaml" DOC=true
```

Comment:
leave all of this

i am not convinced this build is actually reliable enough to change this

Reply:
You'll know better, I just thought that it'd be a good idea to make it break the build for a while, to see if it's actually reliable. Other than taking too long, does it fails with false positives often for any reason?

Reply:
let's make that an independent change.

Reply:
fair enough, reverted that part

Reply:
BTW, the travis build here is only failing when the sphinx build is completely failing, not when there is an error in the doc build. So that shouldn't happen too often.

Reply:
Actually, the doc build *is* failing on travis now. But I am wondering if this is because of "fatal: No such remote: origin". 
In that case, we should either fix that to not try to push the docs the github repo on PRs, or revert this PR, as it will currently always fail on PRs.